testing/README.md

```diff
@@ -12,21 +12,52 @@ Some UIs to check out:
 Testgrid (Test Results Dashboard): https://k8s-testgrid.appspot.com/sig-big-data
 Prow (Test-runner dashboard): https://prow.k8s.io/?repo=tensorflow%2Fminigo
 
-## Local testing
+## Updating continuous integration tests
 
-To test out changes to the docker image, first build the test-harness image:
+Build and push the test-harness image:
 
 ```shell
-make buildv2
+cd testing/
+PROJECT=tensor-go VERSION_TAG=latest make pushv2
 ```
 
-And then run the tests.
+## Test a pull request locally
+
+You can test a PR sent to github locally using the following steps. This
+enables you to poke around if something's failing.
+
+```shell
+export PATH=$HOME/go/bin:$PATH
+cd $HOME  # Or somewhere else outside the minigo repo
+git clone git@github.com:kubernetes/test-infra.git
+cd test-infra/config
+./pj-on-kind.sh pull-tf-minigo-cc
```

Comment:
@cjwagner this is it, you can't ever change the name of your script now ;)

Reply:
🙃 
/shrug